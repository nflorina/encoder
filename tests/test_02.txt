)
K
oBak4^N<$nL7QABTtGeZ.3iw/rmZ6fJYI6m)e+-UXD'yf\n
-1499
';lz)H1psk;*YDpdRF>vN8^+(mNR,q13.
/
-1411
14
j
-13822
.
-13428613
g
o
$hp3"9T,TaR{H=4u[s`q"ef<rJ$S;_E@I6TKO)X%j+!
-13111
-18
-16914
MyE.?fW
$'D\-0h.E2EP8O<0
n&5xT/?bN&:&B?-f{:weI=wom0?+A|z0#0*X@J;oQV
s
$
?SXhqQX`bxl$ug5yx@S9jn)=F1=$m0CBi-4<fu~_c$VL
14562
-145949
c
1912715
b
%
:p'1pgtt?A/qgivG0l
11844716
a
M
R!=mqEb-tXL7g>~R6Fb#XW,TAn?n;=B<[1/!1r.&K[>4z=f2d
>"BsC233MQVj-
z
18365189
-196
\5(B_>`baT'sh:B;q.IY)XSwvT2qsagPwpsX0U;q*Bfs])
-13266
-1274
1
END
