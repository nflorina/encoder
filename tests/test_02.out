11 15 16
))K)n\fy'DXU-+e)m6IYJf6Zmr/wi3.ZeGtTBAQ7Ln$<N^4kaBo1499.31q,RNm(+^8Nv>FRdpDY*';lz)H1psk;)/K1114411j\138221.y131342861gX1oU!+j%X)OKT6I@E_;S$Jr<fe"q`s[u4=H{RaT,T9"3ph$111131814169yE.?fWM0<O8PE2E$'D\-0h.VQo;J@X*0#0z|A+?0mow=n&5xT/?bN&:&B?-f{:weI1si1$GLV$c_~uf<4-iBC0m$=1F=)?SXhqQX`bxl$ug5yx@S9jn621451459491ct91271511b(1%vl0Gvigq/A:p'1pgtt?844716111a>1Mdd2f=z4>[K&.r1!/1[<B=;n?nAT,WX#bF6R~>g7LXt-R!=mqEb-jVQM332CsB">1zP91836518196)]sfB*q;U0XspwPgasq2Tvw\5(B_>`baT'sh:B;q.IY)XS132661274
))K)n\fy'DXU-+e)m6IYJf6Zmr/wi3.ZeGtTBAQ7Ln$<N^4kaBo1499.31q,RNm(+^8Nv>FRdpDY*';lz)H1psk;)/K1114411j\138221.y131342861gX1oU!+j%X)OKT6I@E_;S$Jr<fe"q`s[u4=H{RaT,T9"3ph$111131814169yE.?fWM0<O8PE2E$'D\-0h.VQo;J@X*0#0z|A+?0mow=n&5xT/?bN&:&B?-f{:weI1si1$GLV$c_~uf<4-iBC0m$=1F=)?SXhqQX`bxl$ug5yx@S9jn621451459491ct91271511b(1%vl0Gvigq/A:p'1pgtt?844716111a>1Mdd2f=z4>[K&.r1!/1[<B=;n?nAT,WX#bF6R~>g7LXt-R!=mqEb-jVQM332CsB">1zP91836518196)]sfB*q;U0XspwPgasq2Tvw\5(B_>`baT'sh:B;q.IY)XS132661274
